src/Alpha_complex/test/CMakeLists.txt

```diff
@@ -1,76 +1,76 @@
 project(Alpha_complex_tests)
```

Comment:
I don't think this is what we want to do for this target. Perhaps instead of heavily commenting out lines in the Cmakelists, you make a separate one (+folder) for fuzzing.